nomad/structs/structs.go

```diff
@@ -788,6 +788,9 @@ type DeploymentAllocHealthRequest struct {
 type ApplyDeploymentAllocHealthRequest struct {
 	DeploymentAllocHealthRequest
 
+	// Timestamp is the timestamp to use when setting the allocations health.
```

Comment:
Mention that this is always set by the client, if other attributes can be set by the server.

Reply:
In this case this is not being set by the clients.